### PR TITLE
Yda 5699 add admin page and new prim-group

### PR DESCRIPTION
**Pull request by FuHsinyu**
**Add UI and API endpoints, and associated tests.**
1. Added API: Check and return if the authenticated user has the admin access
2. Added UI: Added an Administration option in dropdown menu for users with admin access, for users without admin access, the option is hidden. 
The new API and UI are associated to yoda-portal PR, please refer to them for the complete story. 
https://github.com/UtrechtUniversity/yoda-portal/commits/YDA-5699-add-admin-page/ 

**Add new priv-admin group to deployment config**
Created a new group "priv-admin" where users being a member of the group are granted with admin access
1. Added script: the updated group-manager-setup.sh checks if priv-admin is already created in irods, if not, the script sets up the priv-admin. 
The new group's deployment is associated to ancible's playbook main.yml and test.yml, referred to the PR of the yoda\s main repo. See
https://github.com/UtrechtUniversity/yoda/tree/YDA-5699-add-admin-page

### CODE REVIEW

**Comment by FuHsinyu** [comment]
Just saw there was a failure in Integration test... will work on it first

File: admin.py [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+    Checks if the user has admin access based on user rights or membership in admin-priv group.
+
+    :param ctx: Combined type of a ctx and rei struct
+
+    :returns: True if the user has the admin access, False otherwise.
+    """
+
+    group_name = "priv-admin"
+
+    # if user has admin right
```
Comment (lwesterhof) [inline admin.py]: Remove debug statements.

File: admin.py [hunk 2]
```diff
@@ -0,0 +1,37 @@
+# -*- coding: utf-8 -*-
+"""Functions for admin access check."""
+
+__copyright__ = 'Copyright 2010-2024, Utrecht University'
```
Comment (lwesterhof) [inline admin.py]: Copyright should be 20204.

File: tests/step_defs/ui/test_ui_admin.py [hunk 3]
```diff
@@ -0,0 +1,51 @@
...
+
+from conftest import portal_url
+
+scenarios("../../features/ui/ui_admin.feature")
+
+
+@when(parsers.parse("user opens link to admin page"))
+def ui_admin_access(browser):
+    url = "{}/admin".format(portal_url)
+    time.sleep(10)  # FIXME: In case of slow VM machine
```
Comment (lwesterhof) [inline tests/step_defs/ui/test_ui_admin.py]: Is this fixed when running against the Docker setup?

File: util/config.py [hunk 4]
```diff
@@ -103,6 +103,7 @@ def __repr__(self):
                 enable_datarequest=False,
                 enable_data_package_archive=False,
                 enable_data_package_download=False,
+                enable_tape_archive=False,  # TODO: REMOVE IT LATER
```
Comment (lwesterhof) [inline util/config.py]: Now is the time to remove it

**Review by lwesterhof — CHANGES_REQUESTED** [review]
Looks good, some minor issues left.

File: tests/step_defs/ui/test_ui_admin.py (hunk 3 above)
Comment (FuHsinyu) [reply tests/step_defs/ui/test_ui_admin.py]: removed

**Review by lwesterhof — APPROVED** [review]
LGTM

**Pull request merged**